templates/nunjucks/_page.njk.html

```diff
@@ -125,6 +125,13 @@ <h1 class="{{ headingClass }}" data-block-id="{{ page.$headingId or page._id }}"
 {% block bodyEnd %}
 {{ super() }}
 <style type="text/css">
+a:link {
```

Comment:
why are these styles not in a separate css file?

Reply:
They should be and they will be.

Long answer: because 
a) I was holding out against the use of sass
b) the organisation of the component dependencies was not stable

Now both of those issues have been resolved and we will be able to rectify